### ISSUE TITLE
Legacy applications theme problem

### ISSUE THREAD

**Issue by skar395** [catppuccin/gtk#92]
**Describe the bug**
Using the theme in "legacy" applications the edge is broken. The problem does not occur with the default theme

**Mention your distribution and your theme version**
Fedora 38, v0.5.0

**Select how you installed the theme**
- [ x] GitHub Releases
- [ ] AUR package 
- [ ] Nixpkg
- [ ] Using the python install script 
- [ ] Flatpak

**Symlinked the gtk-4.0 folder?**
Have you symlinked the `gtk-4.0` folder as mentioned in the [README](https://github.com/catppuccin/gtk/blob/e907775723eb68a5628c1bcd718b9d822cea9aee/README.md?plain=1#L85-L93)? **This is not applicable for users who have installed the theme with the Nixpkg.**
Yes

**Expected behavior**
The border of the application is normal

**Screenshots**
![Schermata del 2023-04-30 14-48-21](https://user-images.githubusercontent.com/50447537/235353935-a2c8f4d1-dbd4-4639-805a-c08f6ca6537c.png)

![Schermata del 2023-04-30 14-48-31](https://user-images.githubusercontent.com/50447537/235353938-a9056605-d5ca-41c5-8b3c-6255a99391ce.png)

**Comment by thrillho93** [comment]
I can confirm I'm running into the same issue on Fedora 38. In fact, it's gotten worse after applying the latest updates. I used the script to install the theme, and did manually symlink gtk-4.0.

**Comment by hyperreal64** [comment]
Looks like this is an upstream issue with Vince Liuice's Colloid theme: https://github.com/vinceliuice/Colloid-gtk-theme/issues/113#issue-1691302327

**Comment by Raiden0456** [comment]
I'm also running into the same problem for the last couple of days, hope this gets fixed soon

**Comment by npv12** [comment]
This issue is due to mutter. This will be fixed when mutter devs fix it. https://gitlab.gnome.org/GNOME/mutter/-/issues/2725

To improve your experience you may try disabling glamor by putting XWAYLAND_NO_GLAMOR=1 in ~/.config/environment.d/00-no-glamor.conf and then restarting your session.

Also worth reading https://github.com/vinceliuice/Colloid-gtk-theme/issues/113#issuecomment-1531146340

**Comment by Raiden0456** [comment]
> This issue is due to mutter. This will be fixed when mutter devs fix it. https://gitlab.gnome.org/GNOME/mutter/-/issues/2725
> 
> To improve your experience you may try disabling glamor by putting XWAYLAND_NO_GLAMOR=1 in ~/.config/environment.d/00-no-glamor.conf and then restarting your session.
> 
> Also worth reading https://github.com/vinceliuice/Colloid-gtk-theme/issues/113#issuecomment-1531146340

Yeah thanks for the info, I've already done this workaround and it does help, but still, I hope it gets fixed soon :/

**Comment by ghostx31** [comment]
I guess since this is an upstream Gnome/Mutter issue, I think we should close this and this should be tracked upstream itself.
